WalletWasabi/WabiSabi/Client/CoinJoinCoinSelector.cs

```diff
@@ -369,9 +369,8 @@ private static IEnumerable<TCoin> AnonScoreTxSourceBiasedShuffle<TCoin>(TCoin[]
 	private static bool TryAddGroup<TCoin>(UtxoSelectionParameters parameters, Dictionary<int, IEnumerable<TCoin>> groups, IEnumerable<TCoin> group)
 		where TCoin : ISmartCoin
 	{
-		var inSum = group.Sum(x => x.EffectiveValue(parameters.MiningFeeRate, parameters.CoordinationFeeRate));
-		var outFee = parameters.MiningFeeRate.GetFee(Constants.P2wpkhOutputVirtualSize);
-		if (inSum >= outFee + parameters.AllowedOutputAmounts.Min)
+		var effectiveInputSum = group.Sum(x => x.EffectiveValue(parameters.MiningFeeRate, parameters.CoordinationFeeRate));
+		if (effectiveInputSum >= parameters.AllowedOutputAmounts.Min)
```

Comment:
Why the outfee was disregarded here?

Reply:
parameters.AllowedOutputAmounts.Min already contains that and more: it now cointains outfee + spending fee for later on